packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/SystemServicesFlutterApiImpl.java

```diff
@@ -8,15 +8,15 @@
 import io.flutter.plugins.camerax.GeneratedCameraXLibrary.SystemServicesFlutterApi;
 
 public class SystemServicesFlutterApiImpl extends SystemServicesFlutterApi {
-  public SystemServicesFlutterApiImpl(
-      BinaryMessenger binaryMessenger, InstanceManager instanceManager) {
+  public SystemServicesFlutterApiImpl(BinaryMessenger binaryMessenger) {
     super(binaryMessenger);
-    this.instanceManager = instanceManager;
   }
 
-  private final InstanceManager instanceManager;
-
-  public void onDeviceOrientationChanged(String orientation, Reply<Void> reply) {
+  public void sendDeviceOrientationChangedEvent(String orientation, Reply<Void> reply) {
     super.onDeviceOrientationChanged(orientation, reply);
   }
+
+  public void sendCameraError(String errorDescription, Reply<Void> reply) {
```

Comment:
Document or add annotations about the expectations of errorDescription and reply?
Nullable/NonNull, can they be empty, etc.